.github/workflows/lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linux:
+    name: "Node ${{ matrix.node }} on Linux: Lint"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
IMO it is a waste to run a commit message validation/lint on every Node version. Do you expect cases when it would fail on one, but not the other? What do you think about making it a job running on the latest Node LTS in the `test.yml` instead?